go.mod

```diff
@@ -3,21 +3,21 @@ module github.com/containerd/nri
 go 1.21
 
 require (
-	github.com/containerd/ttrpc v1.2.6-0.20240827082320-b5cd6e4b3287
+	github.com/containerd/ttrpc v1.2.7
 	github.com/knqyf263/go-plugin v0.8.1-0.20240827022226-114c6257e441
-	github.com/moby/sys/mountinfo v0.6.2
+	github.com/moby/sys/mountinfo v0.7.2
 	github.com/onsi/ginkgo/v2 v2.19.1
 	github.com/onsi/gomega v1.34.0
-	github.com/opencontainers/runtime-spec v1.0.3-0.20220825212826-86290f6a00fb
+	github.com/opencontainers/runtime-spec v1.1.0
```

Comment:
<details>
<summary>Release notes</summary>
<p><em>Sourced from <a href="https://github.com/opencontainers/runtime-spec/releases">github.com/opencontainers/runtime-spec's releases</a>.</em></p>
<blockquote>
<h2>v1.1.0</h2>
<p>Vote: <a href="https://redirect.github.com/opencontainers/runtime-spec/pull/1213">opencontainers/runtime-spec#1213</a></p>
<p>Blog: <a href="https://opencontainers.org/posts/blog/2023-07-21-oci-runtime-spec-v1-1/">https://opencontainers.org/posts/blog/2023-07-21-oci-runtime-spec-v1-1/</a></p>
<h3>Breaking changes (but rather conforms to the existing runc implementation)</h3>
<ul>
<li>config: change prestart hook spec to match reality (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1169">#1169</a>)</li>
</ul>
<h3>Deprecations</h3>
<ul>
<li>config-linux: mark memory.kernel[TCP] as NOT RECOMMENDED (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1093">#1093</a>)</li>
</ul>
<h3>Additions</h3>
<ul>
<li>cgroup: add cgroup v2 support (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1040">#1040</a>)</li>
<li>seccomp: allow to override errno return code (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1041">#1041</a>)</li>
<li>seccomp: Add support for SCMP_ACT_KILL_PROCESS (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1044">#1044</a>)</li>
<li>Update seccomp architectures to support RISCV64 (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1059">#1059</a>)</li>
<li>Add support for SCMP_ACT_KILL_THREAD (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1064">#1064</a>)</li>
<li>Add Seccomp Notify support using UNIX sockets and container metadata (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1074">#1074</a>)</li>
<li>config-linux: Add Intel RDT CMT and MBM Linux support (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1076">#1076</a>)</li>
<li>seccomp: allow to override default errno return code (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1087">#1087</a>)</li>
<li>Introduce zos as platform (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1095">#1095</a>)</li>
<li>config-linux: add idle option for container cgroup (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1136">#1136</a>)</li>
<li>config-linux: add CFS bandwidth burst (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1120">#1120</a>)</li>
<li>IDMapping field for mount point (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1143">#1143</a>)</li>
<li>schema: add cpu idle (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1145">#1145</a>)</li>
<li>add domainname spec entity (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1156">#1156</a>)</li>
<li>config-linux: add memory.checkBeforeUpdate (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1158">#1158</a>)</li>
<li>seccomp: Add flag SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV (<a href="https://redirect.github.com/opencontainers/runtime-spec/issues/1161">#1161</a>)</li>
</ul>
<!-- raw HTML omitted -->
</blockquote>
<p>... (truncated)</p>
</details>
<details>
<summary>Commits</summary>
<ul>
<li>See full diff in <a href="https://github.com/opencontainers/runtime-spec/commits/v1.1.0">compare view</a></li>
</ul>
</details>